install/agones/README.md

```diff
@@ -0,0 +1,83 @@
+
+| Parameter                            | Description                               | Default                                              |
+| ------------------------------------ | ----------------------------------------- | ---------------------------------------------------- |
+| `namespace`                          | Namespace to use for Agones               | `agones-system`                                      |
+| `serviceaccount.controller`          | Service account name for the controller   | `agones-controller`                                  |
+| `serviceaccount.sdk`                 | Service account name for the sdk          | `agones-sdk`                                         |
+| `image.controller.repository`        | Image repository for the controller       | `gcr.io/agones-images/agones-controller`             |
+| `image.controller.tag`               | Image tag for the controller              | `0.1`                                                |
+| `image.controller.pullPolicy`        | Image pull policy for the controller      | `IfNotPresent`                                       |
+| `image.sdk.repository`               | Image repository for the sdk              | `gcr.io/agones-images/agones-sdk`                    |
+| `image.sdk.tag`                      | Image tag for the sdk                     | `0.1`                                                |
+| `image.sdk.alwaysPull`               | Tells if the sdk image should always be pulled  | `false`                                        |
+| `minPort`                            | Minimum port to use for dynamic port allocation | `7000` |
+| `maxPort`                            | Maximum port to use for dynamic port allocation | `8000` |
+| `healthCheck.http.port`              | Port to use for liveness probe service          | `8080` |
+| `healthCheck.initialDelaySeconds`    | Initial delay before performing the first probe (in seconds) | `3` |
+| `healthCheck.periodSeconds`          | Seconds between every liveness probe (in seconds)   | `3` |
+| `healthCheck.failureThreshold`       | Number of times before giving up (in seconds)         | `3` |
+| `healthCheck.timeoutSeconds`         | Number of seconds after which the probe times out (in seconds)         | `1` |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --name my-release \
+  --set namespace=mynamespace,minPort=1000,maxPort=5000 agones
+```
+
+The above command sets the namespace where Agones is deployed to `mynamespace`. Additionally Agones will use a dynamic port allocation range of 1000-5000.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml agones
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
```

Comment:
At the end of the documentation, should we redirect them back to the "## Confirming Agones started successfully" section of `docs/installing_agones.md` ? otherwise they may not catch the "what to do next" section? WDYT?